### PR TITLE
adjusted spacing in navbar for mobile

### PR DESCRIPTION
**Pull request by khanh-do**
### Describe the problem being solved:
- [x] increase space between the bottom of the hamburger icon and first item in the dropdown
- [x] decrease space between items in the drop down list

### Impacted areas in the application: 
* navbar

List general components of the application that this PR will affect: 
* common

Screenshot from 2019-10-23 20-20-20

Screenshot from 2019-10-23 20-23-25

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

### CODE REVIEW

**Comment by khanh-do** [comment]
I can adjust the spacing further if needed. Please let me know.

File: frontend/src/components/common/header/index.css [hunk 1]
```diff
@@ -183,4 +183,13 @@ a.dropdown-item {
   .donate {
     font-size: 50px;
   }
+
+  .resources-link {
+    padding: 5px 8px;
```
Comment (dkaplan88) [inline frontend/src/components/common/header/index.css]: lets just do `0px 8px` here

File: frontend/src/components/common/header/index.css [hunk 2]
```diff
@@ -183,4 +183,13 @@ a.dropdown-item {
...
+  }
+
+  .align-links {
+    padding-top: 35px;
```
Comment (dkaplan88) [inline frontend/src/components/common/header/index.css]: this can also be `15px`

File: frontend/src/components/common/header/index.css (hunk 1 above)
Comment (khanh-do) [reply frontend/src/components/common/header/index.css]: Thank you, Dan. I made the change.

File: frontend/src/components/common/header/index.css (hunk 2 above)
Comment (khanh-do) [reply frontend/src/components/common/header/index.css]: Done.

**Pull request merged**